Explain F-Droid Anti-Features (#37)
tailscale/tailscale-android: metadata/en-US/full_description.txt

Tailscale is a mesh VPN alternative that makes it easy to connect your devices, wherever they are. No more fighting configuration or firewall ports. Built on WireGuard®, Tailscale enables an incremental shift to zero-trust networking by implementing “always-on” remote access. This guarantees a consistent, portable, and secure experience independent of physical location.

WireGuard is a registered trademark of Jason A. Donenfeld.

> Anti-Features:

Reviewer: ```suggestion
F-Droid Anti-Features:
```

> * NonFreeNet - uses a non-free network service which is impossible, or not easy to replace without rebuilding the app

Reviewer: ```suggestion
* NonFreeNet - Tailscale provides a <a href="https://tailscale.com/pricing/">Free plan for up to 3 users</a>. <a href="https://github.com/juanfont/headscale/blob/main/docs/android-client.md">Headscale</a> provides a self-hosted alternative.
```

Author: Not sure I follow, this is about Tailscale, whatever else exists out there is not related here.

Reviewer 2: The client can be switched from using the Tailscale hosted control plane to a self-hosted control plane using Headscale. The link points to the instructions on how to switch the app from using the hosted control plane to a self-hosted instance.

Author: Right, did not open the link on mobile :+1:

> * Tracking - logs and reports debugging information continously to the app developers, see <a href="https://gitlab.com/fdroid/fdroiddata/-/issues/2601">F-Droid Data Issue #2601</a>

Reviewer: ```suggestion
* Tracking - Tailscale reports logging information to our central logging server that is used to provide support, troubleshooting, and administrative features. <a href="https://tailscale.com/kb/1011/log-mesh-traffic/#opting-out-of-client-logging">Client logs can be disabled on some platforms</a>, but not yet Android. For more information on what information is logged and how it is used, see <a href="https://tailscale.com/kb/1011/log-mesh-traffic/">an explanation of logs</a> and <a href="https://tailscale.com/privacy-policy/#the-information-we-collect">Tailscale's privacy policy</a>. For tracking changes to this, see <a href="https://gitlab.com/fdroid/fdroiddata/-/issues/2601">F-Droid Data Issue #2601</a>.
```
